profiles/rhel/prepare-system

```diff
@@ -0,0 +1,62 @@
+INT_CERTS="${INT_CERTS:-http://download.eng.bos.redhat.com/brewroot/vol/rhel-7/packages/redhat-internal-cert-install/0.1/9.el7.csb/noarch/redhat-internal-cert-install-0.1-9.el7.csb.noarch.rpm}"
+
+if ! rpm -q redhat-internal-cert-install >/dev/null 2>&1; then
+    echo "Deploy internal certs."
+    yum install -y "$INT_CERTS"
```

Comment:
Should we validate against $? from the yum install? I assume the internal cert install failing would be a failure condition.

Reply:
I am actually just moving this code from `mtps-prepare-script` to a separate file. So this code is not new.

But to answer your question: `prepare-system` is injected into `mtps-prepare-system`: https://github.com/fedora-ci/mini-tps/blob/8c1a2e78c61c1128b24da10fdd9af9f46755a181/mtps-prepare-system#L153
And that script has `-e` (fail on any non-zero return value) in shebang: `#!/bin/bash -efu`

Reply:
Okay, that makes sense. Thanks!